cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_key.cli

```diff
@@ -1,6 +1,8 @@
 Usage: cardano-cli allegra key 
                                  ( verification-key
                                  | non-extended-key
+                                 | generate-mnemonic
+                                 | key-from-mnemonic
```

Comment:
This command feels a little odd: `cardano-cli latest key key-from-mnemonic`

I think this key prefix seems redundant.
```suggestion
                                 | from-mnemonic
```

Reply:
Or we can call it `derive-from-mnemonic`

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/975/commits/e0c808d8e2c0d6b438905c6efcca2205f497c0e6